lib/nexpose/filter.rb

```diff
@@ -120,8 +120,8 @@ module Field
 
       # Search based on the last scan date of an asset.
       # Valid Operators: ON_OR_BEFORE, ON_OR_AFTER, BETWEEN, EARLIER_THAN, WITHIN_THE_LAST
-      # Valid Values: Use Value::ScanDate::FORMAT for date arguments.
-      #               Use FixNum for day arguments.
+      # Valid Values: Use FixNum for EARLIER_THAN and WITHIN_THE_LAST.
```

Comment:
Should probably explain why it needs to be a fixnum instead of a date value since it's not spelled out otherwise in the code docs. Just to give a little extra help to newcomers to this stuff.